[WIP] Added routes for creating and triggering builds

jhellar opened the pull request:
# Motivation
https://issues.jboss.org/browse/AEROGEAR-7911
https://issues.jboss.org/browse/AEROGEAR-7915

wojta commented on pkg/web/buildConfigsHandler.go:
```diff
@@ -3,27 +3,182 @@ package web
+	}
+}
+
+func newBuildConfigObject(data BuildConfigCreateRequest, namespace string) (*mobile.BuildConfig, []v1.Secret) {
```
This would be better to split into more functions, it's hard to read.

jhellar commented:
So that ci check can complete I've moved this branch to repo in aerogear org. I've also addressed @wojta's comments and added tests. https://github.com/aerogear/mobile-client-service/pull/55
